grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/StatusMessageEscaper.java

```diff
@@ -102,10 +105,8 @@ private static String doEscape(byte[] valueBytes, int ri) {
             }
             escapedBytes[wi++] = b;
         }
-        final byte[] dest = new byte[wi];
-        System.arraycopy(escapedBytes, 0, dest, 0, wi);
 
-        return new String(dest, StandardCharsets.US_ASCII);
+        return new String(escapedBytes, 0, wi, StandardCharsets.US_ASCII);
```

Comment:
@anuraaga Is it OK to use `new String(escapedBytes, 0, 0, wi)` instead of this for even better performance? It seems so, but just in case :-)

And please note that I removed an extra byte array allocation :stuck_out_tongue_winking_eye: